app/models/user.rb

```diff
@@ -5,4 +5,9 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
   has_many :rentals
   has_many :items, through: :rentals
+
+  validates :first_name, :last_name, :username, :adress, :encrypted_password, :email, presence: true
```

Comment:
address is misspelt, i would suggest taking these names from the schema so you don't make mistakes

`validates :username, :first_name, :last_name, :email, :address, :encrypted_password, presence: true
end`